ext/ITensorNetworksEinExprsExt.jl

```diff
@@ -0,0 +1,37 @@
+module ITensorNetworksEinExprsExt
+
+using ITensorNetworks
+using ITensorNetworks: Index
+using EinExprs: EinExprs, EinExpr, einexpr, SizedEinExpr
+
+function ITensorNetworks.contraction_sequence(
+  ::ITensorNetworks.Algorithm"einexpr",
+  tn::ITensorNetwork{T};
+  optimizer=EinExprs.Exhaustive(),
+) where {T}
+  IndexType = Any
+
+  tensors = EinExpr{IndexType}[]
+  tensor_map = Dict{Set{IndexType},T}()
+  sizedict = Dict{IndexType,Int}()
+
+  for (key, tensor) in pairs(vertex_data(tn))
+    _inds = collect(ITensorNetworks.inds(tensor))
+    push!(tensors, EinExpr{IndexType}(; head=_inds))
+    tensor_map[Set(_inds)] = key
+    merge!(sizedict, Dict(_inds .=> size(tensor)))
+  end
+
+  _openinds = collect(ITensorNetworks.externalinds(tn))
```

Comment:
`ITensorNetworks.externalinds() -> externalinds()`

Reply:
I don't think it is automatically exported.
```julia
julia> using ITensorNetworks

julia> externalinds
ERROR: UndefVarError: `externalinds` not defined
```

Reply:
Strange, are you on `ITensorNetworks 0.4.0`? This should have been added with #123 .

Reply:
Nope, I'm on v0.3.10. Let me rebase then.